app/services/big_blue_button_api.rb

```diff
@@ -7,17 +7,18 @@ def initialize; end
 
   # Sets a BigBlueButtonApi object for interacting with the API.
   def bbb_server
+    # TODO: Amir - Protect the BBB secret.
     # TODO: Hadi - Add additional logic here...
     @bbb_server ||= BigBlueButton::BigBlueButtonApi.new(bbb_endpoint, bbb_secret, '1.8')
   end
 
   private
 
   def bbb_endpoint
-    ENV['BIGBLUEBUTTON_ENDPOINT']
+    ENV['BIGBLUEBUTTON_ENDPOINT'] || 'https://test-install.blindsidenetworks.com/bigbluebutton/api'
```

Comment:
Lets replace this with `ENV.fetch(..., ...)`

Reply:
Alright,

Reply:
DONE,